src/main/kotlin/com/dprint/listeners/OnSaveListener.kt

```diff
@@ -1,39 +0,0 @@
-package com.dprint.listeners
-
-import com.dprint.config.ProjectConfiguration
-import com.dprint.config.UserConfiguration
-import com.dprint.core.Bundle
-import com.dprint.core.LogUtils
-import com.dprint.services.FormatterService
-import com.intellij.openapi.components.service
-import com.intellij.openapi.diagnostic.logger
-import com.intellij.openapi.fileEditor.FileEditorManager
-import com.intellij.openapi.project.Project
-import com.intellij.openapi.vfs.newvfs.BulkFileListener
-import com.intellij.openapi.vfs.newvfs.events.VFileEvent
-
-private val LOGGER = logger<OnSaveListener>()
-
-/**
- * A listener that looks for events coming from saves. It attempts to format the content of the saved file with Dprint.
- */
-class OnSaveListener(private val project: Project) : BulkFileListener {
```

Comment:
Basically this is a pretty low level operation and obviously gets triggered when the external formatter is run.